src/browser/components/atoms/Userimage/userImage.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+import styles from './userimage.css'
+export default class UserImage extends Component{
+  render() {
+    return (
+      <div>
+      <img className='user-image' src='http://placehold.it/50x50&text=slide1' />
```

Comment:
Are you use you need this div? If you do consider moving the `className='user-image'` to the div.